server/python/plugin_remote.py

```diff
@@ -1043,6 +1043,11 @@ async def plugin_async_main(
         peer, api, pluginId, hostInfo, loop
     )
 
+    async def createMediaManager():
+        return MediaManager(await peer.getParam("getMediaManager"))
```

Comment:
I think it should likely return the object as is, without the wrapper class. the caller will wrap it.